nrf52832-hal/Cargo.toml

```diff
@@ -40,5 +40,7 @@ version = "0.2.1"
 [features]
 doc = []
 rt = ["nrf52832-pac/rt"]
-default = ["rt"]
+xxAA-package = []
+xxAB-package = []
+default = ["rt", "xxAA-package"]
```

Comment:
I think this should default to `xxAB-package`. That has the smaller memory, which makes it the safe choice.

Reply:
Hm, I see what you mean, however the xxAA package is *much* more widely used, particularly on development boards. This would put the inconvenience on the majority of users.

If you feel strongly about this, I can make the change, however I would prefer not to.

Reply:
I still feel the same, but @hannobraun's opinion and @simonsso's :+1: have convinced me :)

Change made.

Reply:
I was just about to say that I don't feel strongly enough to overrule you. Better merge this fast, before you change your mind again ;-)